hal/src/b5som/network/network.cpp

```diff
@@ -152,6 +152,23 @@ int if_init_platform(void*) {
     return 0;
 }
 
+int if_init_platform_postpone(void*) {
+    if (en2) {
+        uint8_t dummy;
+        if (!if_get_index(en2->interface(), &dummy)) {
+            auto wizNetif = reinterpret_cast<WizNetif*>(en2);
+            if (wizNetif->isPresent()) {
+                return 0;
+            }
+            netifapi_netif_remove(en2->interface());
```

Comment:
It did not keep the interface in the list previously if it is invalid, since it was not even successfully add to the list. It just reserves the index if failed to add the interface.